modules/services/window-managers/fluxbox.nix

```
@@ -0,0 +1,116 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.xsession.windowManager.fluxbox;
+in {
+  meta.maintainers = [ maintainers.AndersonTorres ];
+
+  options = {
+    xsession.windowManager.fluxbox = {
+      enable = mkEnableOption "Fluxbox window manager";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.fluxbox;
+        defaultText = literalExpression "pkgs.fluxbox";
+        description = "Package to use for running Fluxbox WM.";
+      };
+
+      init = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Init configuration for Fluxbox, written to <filename>~/.fluxbox/init</filename>. Look at the <citerefentry><refentrytitle>fluxbox</refentrytitle><manvolnum>1</manvolnum></citerefentry> manpage for details.
```

Best to avoid very long lines, can do something like

```suggestion
          Init configuration for Fluxbox, written to
          <filename>~/.fluxbox/init</filename>. See
          <citerefentry>
            <refentrytitle>fluxbox</refentrytitle>
            <manvolnum>1</manvolnum>
          </citerefentry>
          for details.
```